The bot fails with an error compiling the dispatch overlay, which might suffer from the same issue (perhaps it's finding the system overlay dylibs rather than the just-built ones):

```
15:19:02 cd /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.0-branch/buildbot_incremental/swift-macosx-x86_64/stdlib/public/SDK/Dispatch && /usr/bin/python /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.0-branch/swift/utils/line-directive @/Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.0-branch/buildbot_incremental/swift-macosx-x86_64/stdlib/public/SDK/Dispatch/QrFmE.txt -- /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.0-branch/buildbot_incremental/swift-macosx-x86_64/./bin/swiftc -c -sdk /Applications/Xcode-beta.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.0.sdk -target arm64-apple-ios7.0 -resource-dir /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.0-branch/buildbot_incremental/swift-macosx-x86_64/./lib/swift -F /Applications/Xcode-beta.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.0.sdk/../../../Developer/Library/Frameworks -O -D INTERNAL_CHECKS_ENABLED -D SWIFT_ENABLE_RUNTIME_FUNCTION_COUNTERS -I /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.0-branch/buildbot_incremental/swift-macosx-x86_64/./lib/swift/iphoneos/arm64 -module-cache-path /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.0-branch/buildbot_incremental/swift-macosx-x86_64/./module-cache -no-link-objc-runtime -Xfrontend -sil-verify-all -Xfrontend -enable-resilience -Xfrontend -enable-sil-ownership -Xfrontend -enable-mandatory-semantic-arc-opts -Xfrontend -enforce-exclusivity=unchecked -swift-version 5 -swift-version 4 -autolink-force-load -warn-swift3-objc-inference-complete -Xfrontend -verify-syntax-tree -warn-implicit-overrides -Fsystem /Applications/Xcode-beta.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.0.sdk/System/Library/PrivateFrameworks/ -module-link-name swiftDispatch -force-single-frontend-invocation -parse-as-library -o /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.0-branch/buildbot_incremental/swift-macosx-x86_64/stdlib/public/SDK/Dispatch/iphoneos/arm64/Dispatch.o @/Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.0-branch/buildbot_incremental/swift-macosx-x86_64/stdlib/public/SDK/Dispatch/QrFmE.txt
15:19:02 
/Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.0-branch/swift/stdlib/public/SDK/Dispatch/Queue.swift:100:108: error: missing argument label 'rawValue:' in call
15:19:02                                         return __dispatch_queue_attr_make_with_autorelease_frequency(attr, __dispatch_autorelease_frequency_t(0))
15:19:02                                                                                                                                               ^
15:19:02                                                                                                                                               rawValue: 
15:19:02 /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.0-branch/swift/stdlib/public/SDK/Dispatch/Queue.swift:100:73: error: value of optional type '__dispatch_autorelease_frequency_t?' must be unwrapped to a value of type '__dispatch_autorelease_frequency_t'
15:19:02                                         return __dispatch_queue_attr_make_with_autorelease_frequency(attr, __dispatch_autorelease_frequency_t(0))
15:19:02                                                                                                            ^
15:19:02 /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.0-branch/swift/stdlib/public/SDK/Dispatch/Queue.swift:100:73: note: coalesce using '??' to provide a default when the optional value contains 'nil'
15:19:02                                         return __dispatch_queue_attr_make_with_autorelease_frequency(attr, __dispatch_autorelease_frequency_t(0))
15:19:02                                                                                                            ^
15:19:02                                                                                                                                                  ?? <#default value#>
15:19:02 /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.0-branch/swift/stdlib/public/SDK/Dispatch/Queue.swift:100:73: note: force-unwrap using '!' to abort execution if the optional value contains 'nil'
15:19:02                                         return __dispatch_queue_attr_make_with_autorelease_frequency(attr, __dispatch_autorelease_frequency_t(0))
15:19:02                                                                                                            ^
15:19:02                                                                                                                                                 !
15:19:02 /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.0-branch/swift/stdlib/public/SDK/Dispatch/Queue.swift:103:108: error: missing argument label 'rawValue:' in call
15:19:02                                         return __dispatch_queue_attr_make_with_autorelease_frequency(attr, __dispatch_autorelease_frequency_t(1))
15:19:02                                                                                                                                               ^
15:19:02                                                                                                                                               rawValue: 
15:19:02 /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.0-branch/swift/stdlib/public/SDK/Dispatch/Queue.swift:103:73: error: value of optional type '__dispatch_autorelease_frequency_t?' must be unwrapped to a value of type '__dispatch_autorelease_frequency_t'
15:19:02                                         return __dispatch_queue_attr_make_with_autorelease_frequency(attr, __dispatch_autorelease_frequency_t(1))
15:19:02                                                                                                            ^
15:19:02 /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.0-branch/swift/stdlib/public/SDK/Dispatch/Queue.swift:103:73: note: coalesce using '??' to provide a default when the optional value contains 'nil'
15:19:02                                         return __dispatch_queue_attr_make_with_autorelease_frequency(attr, __dispatch_autorelease_frequency_t(1))
15:19:02                                                                                                            ^
15:19:02                                                                                                                                                  ?? <#default value#>
15:19:02 /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.0-branch/swift/stdlib/public/SDK/Dispatch/Queue.swift:103:73: note: force-unwrap using '!' to abort execution if the optional value contains 'nil'
15:19:02                                         return __dispatch_queue_attr_make_with_autorelease_frequency(attr, __dispatch_autorelease_frequency_t(1))
15:19:02                                                                                                            ^
15:19:02                                                                                                                                                 !
15:19:02 /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.0-branch/swift/stdlib/public/SDK/Dispatch/Queue.swift:106:108: error: missing argument label 'rawValue:' in call
15:19:02                                         return __dispatch_queue_attr_make_with_autorelease_frequency(attr, __dispatch_autorelease_frequency_t(2))
15:19:02                                                                                                                                               ^
15:19:02                                                                                                                                               rawValue: 
15:19:02 /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.0-branch/swift/stdlib/public/SDK/Dispatch/Queue.swift:106:73: error: value of optional type '__dispatch_autorelease_frequency_t?' must be unwrapped to a value of type '__dispatch_autorelease_frequency_t'
15:19:02                                         return __dispatch_queue_attr_make_with_autorelease_frequency(attr, __dispatch_autorelease_frequency_t(2))
15:19:02                                                                                                            ^
15:19:02 /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.0-branch/swift/stdlib/public/SDK/Dispatch/Queue.swift:106:73: note: coalesce using '??' to provide a default when the optional value contains 'nil'
15:19:02                                         return __dispatch_queue_attr_make_with_autorelease_frequency(attr, __dispatch_autorelease_frequency_t(2))
15:19:02                                                                                                            ^
15:19:02                                                                                                                                                  ?? <#default value#>
15:19:02 /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.0-branch/swift/stdlib/public/SDK/Dispatch/Queue.swift:106:73: note: force-unwrap using '!' to abort execution if the optional value contains 'nil'
15:19:02                                         return __dispatch_queue_attr_make_with_autorelease_frequency(attr, __dispatch_autorelease_frequency_t(2))
15:19:02                                                                                                            ^
15:19:02                                                                                                                                                 !
```

--- a/test/SIL/Serialization/deserialize_coregraphics.swift
+++ b/test/SIL/Serialization/deserialize_coregraphics.swift
@@ -2,6 +2,7 @@
 // RUN: %target-sil-opt -assume-parsing-unqualified-ownership-sil %platform-sdk-overlay-dir/CoreGraphics.swiftmodule > /dev/null
 // RUN: llvm-bcanalyzer %platform-sdk-overlay-dir/CoreGraphics.swiftmodule | %FileCheck %s
 
+// REQUIRES: rdar51868140
 // REQUIRES: objc_interop
 
 // CHECK-NOT: Unknown

--- a/test/SIL/Serialization/deserialize_darwin.sil
+++ b/test/SIL/Serialization/deserialize_darwin.sil
@@ -2,6 +2,7 @@
 // RUN: %target-sil-opt -assume-parsing-unqualified-ownership-sil %platform-sdk-overlay-dir/Darwin.swiftmodule > /dev/null
 // RUN: llvm-bcanalyzer %platform-sdk-overlay-dir/Darwin.swiftmodule | %FileCheck %s
 
+// REQUIRES: rdar51868140
 // REQUIRES: objc_interop
 
 // CHECK-NOT: Unknown

--- a/test/SIL/Serialization/deserialize_objectivec.sil
+++ b/test/SIL/Serialization/deserialize_objectivec.sil
@@ -2,6 +2,7 @@
 // RUN: %target-sil-opt -assume-parsing-unqualified-ownership-sil %platform-sdk-overlay-dir/ObjectiveC.swiftmodule > /dev/null
 // RUN: llvm-bcanalyzer %platform-sdk-overlay-dir/ObjectiveC.swiftmodule | %FileCheck %s
 
+// REQUIRES: rdar51868140
 // REQUIRES: objc_interop
 
 // CHECK-NOT: Unknown

--- a/test/SIL/Serialization/deserialize_stdlib.sil
+++ b/test/SIL/Serialization/deserialize_stdlib.sil
@@ -2,4 +2,6 @@
 // RUN: %target-sil-opt -assume-parsing-unqualified-ownership-sil %platform-module-dir/Swift.swiftmodule > /dev/null
 // RUN: llvm-bcanalyzer %platform-module-dir/Swift.swiftmodule | %FileCheck %s
 
+// REQUIRES: rdar51868140
+
 // CHECK-NOT: Unknown

--- a/validation-test/execution/interpret-with-dependencies.swift
+++ b/validation-test/execution/interpret-with-dependencies.swift
@@ -1,3 +1,4 @@
+// REQUIRES: rdar51868140
 // REQUIRES: OS=macosx
 // RUN: %empty-directory(%t)
 

